sql/hive/src/main/scala/org/apache/spark/sql/hive/TestHive.scala

```diff
@@ -68,6 +68,8 @@ class TestHiveContext(sc: SparkContext) extends HiveContext(sc) {
   System.clearProperty("spark.hostPort")
   CommandProcessorFactory.clean(hiveconf)
 
+  hiveconf.set("hive.plan.serialization.format", "javaXML")
```

Comment:
Wait, actually?  Should this be in `configure()` so it gets rerun when we do a `RESET`?